tasks/uninstall-transport-ssh.yml

```diff
@@ -0,0 +1,19 @@
+---
+- set_fact:
+    ssh_config_path: "/var/lib/vdsm/.ssh"
+    ssh_config_owner: "vdsm"
+  when:
+    - "v2v_host_type == 'rhv'"
```

Comment:
We should not do that unless we're sure the key was created by install role. It is normal to have SSH key configured on VDSM host for other reasons and we cannot remove it arbitrarily.